app/lib/methods/handleImageBase64.test.ts

```diff
@@ -0,0 +1,49 @@
+import { sanitizeLikeString } from '../database/utils';
+import { getBase64MimeType, isImageBase64, valueOfFirst120CharactersOfImageBase64 } from './handleBase64';
+
+const base64 =
```

Comment:
If the image is the same, why is it duplicated in the code?
Creates a constant with this image in another file.

Reply:
IMHO isn't needed, because I will need to export a const from a `file.test`, right? And this is just a small data, inside 2 different tests files